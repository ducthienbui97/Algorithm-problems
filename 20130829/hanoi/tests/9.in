3
9
AB AC BC CA BA CB
9
AC CB BC AB CA BA
9
AB CA AC CB BA BC
