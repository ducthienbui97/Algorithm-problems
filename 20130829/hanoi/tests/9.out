511
6561
13121
